### DISCUSSION TITLE
Can you export exact timestamp of a pageview?

### DISCUSSION THREAD

**Discussion by Adawg4** [plausible/analytics#1873] [Anything Else]
Looked at the docs for the Stats API and don't know if that's where you get it. Really need one stat of that timestamp for users that hit my site that day. Im trying to see if users hit at (example time: 2:50 and it was counted as 3pm). This is of course while respecting privacy, it's for a range of users, and im trying to provide support to them + merge the hits with support tickets reported times of a bug happening.

**Comment by metmarkosaric** [comment]
This is not possible unfortunately. We don't have granular views like that.
